cmd/airflow.go

```diff
@@ -827,7 +827,7 @@ func airflowPytest(cmd *cobra.Command, args []string) error {
 
 	buildSecretString = util.GetbuildSecretString(buildSecrets)
 
-	exitCode, err := containerHandler.Pytest(pytestFile, customImageName, "", buildSecretString, pytestArgs)
+	exitCode, err := containerHandler.Pytest(pytestFile, customImageName, "", pytestArgs, buildSecretString)
```

Comment:
We should validate the input in mock in our tests, instead of doing mock.Anything